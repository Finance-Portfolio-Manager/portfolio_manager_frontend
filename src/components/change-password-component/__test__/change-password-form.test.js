import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import ChangePasswordForm from './../change-password-form';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = shallow(<ChangePasswordForm></ChangePasswordForm>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<ChangePasswordForm></ChangePasswordForm>).toJSON();
    expect(tree).toMatchSnapshot();
})