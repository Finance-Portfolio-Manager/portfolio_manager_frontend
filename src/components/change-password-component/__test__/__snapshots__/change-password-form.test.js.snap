// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
  data-testid="login-form"
>
  <form
    className=""
  >
    <div
      className="aroundLogo"
    >
      <img
        alt="revature logo"
        className="logo"
        src="revlogo.jpg"
      />
    </div>
    <div
      className="form-inner"
    >
      <h4>
        Portfolio Manager
      </h4>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formBasicEmail"
        >
          Email:
        </label>
        <input
          className="ChangePasswordInput form-control"
          id="formBasicEmail"
          name="email"
          placeholder="Enter your Email"
        />
      </div>
      <button
        className="submitButton mt-2"
        type="submit"
      >
        Get Code
      </button>
      <h5 />
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formBasicCode"
        >
          Confirmation Code:
        </label>
        <input
          className="ChangePasswordInput form-control"
          id="formBasicCode"
          name="code"
          placeholder="Enter your code"
          type="number"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formBasicPassword"
        >
          Password:
        </label>
        <input
          className="ChangePasswordInput form-control"
          id="formBasicPassword"
          name="password"
          placeholder="Enter your password"
          type="password"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="form-confirm-password"
        >
          Confirm Password:
        </label>
        <input
          className="form-control"
          id="form-confirm-password"
          name="confirmPassword"
          placeholder="Re-enter your password"
          required={true}
          type="password"
        />
      </div>
      <button
        className="submitButton mt-2"
        type="submit"
      >
        Confirm
      </button>
      <h5 />
    </div>
  </form>
</div>
`;
